File: output/qwen2.5-coder-32b-instruct-fp16/code_snippet_32.cs
using System;

class Program
{
    static int Factorial(int n)
    {
        if (n == 0) return 1;
        int result = 1;
        for (int i = 1; i <= n; i++)
            result *= i;
        return result;
    }

    static void Main()
    {
        Console.WriteLine(Factorial(5));  // Output: 120
    }
}

File: output/nhn-small-latest/code_go/code_snippet_32.go
package main

import (
    "fmt"
)

func factorial(n int) int {
    if n <= 1 {
        return 1
    }
    return n * factorial(n-1)
}

func main() {
    fmt.Printf("Factorial of 5 is %d\n", factorial(5))
}